packages/terra-clinical-item-collection/tests/jest/_ListView.test.jsx

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import ItemCollection from '../../src/ItemCollection';
 import ListView from '../../src/_ListView';
 
-const defaultRequiredElements = { hasStartAccessory: true, numberOfDisplays: 8, hasComment: true, hasEndAccessory: true };
+const defaultRequiredElements = {
+  hasStartAccessory: true, numberOfDisplays: 8, hasComment: true, hasEndAccessory: true,
```

Comment:
Should we have a new line return for each key?

Reply:
Ask eslint, this is what eslint fix does by default.